spec/admin/views/quizzes/index_spec.rb

```diff
@@ -0,0 +1,13 @@
+require 'spec_helper'
+require_relative '../../../../apps/admin/views/quizzes/index'
+
+describe Admin::Views::Quizzes::Index do
+  let(:exposures) { Hash[foo: 'bar'] }
+  let(:template)  { Lotus::View::Template.new('apps/admin/templates/quizzes/index.html.erb') }
```

Comment:
Line is too long. [94/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.